### PR TITLE
chore: Bump ceramic deps

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,6 +55,7 @@
     "http-status-codes": "^2.1.4",
     "ipfs-core": "^0.3.0",
     "ipfs-http-client": "^48.1.2",
+    "key-did-resolver": "^1.1.2-rc.3",
```
Comment (oed) [inline package.json:58]: I assume this pkg is not needed since it's not imported anywhere?

**Review by oed — APPROVED** [review]
lgtm, would like to get a ✅ from @stbrody as well though.

File: package.json (hunk 1 above)
Comment (ukstv) [reply package.json:58]: as a bunch of other packages

Comment (oed) [reply package.json:58]: What do you mean?

Comment (ukstv) [reply package.json:58]: base64url, config.ts, colors, ejs and few others are not used. Have a separate PR in the making to drop them.

Comment (ukstv) [reply package.json:58]: https://github.com/ceramicnetwork/ceramic-anchor-service/pull/377 @oed

**Review by stbrody — APPROVED** [review]
:rocket:

**Pull request merged**